authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,186 @@
+
+### Ethereum
+
+In Ethereum 2.0 only the validators are incentivized. They are given information by beacon nodes which will probably be run by large providers and used remotely (beacon nodes with data for all shards will need above-average machines) and they build crosslinks (see above) and attest to other crosslinks from that data. To become a validator in Ethereum, one needs 32 ether to stake - no more, no less.
+
+In Ethereum, the less Ether is staked the higher the return rate, to incentivize more people to join. For example, for 1 million ether, the annual return rate is predicted to be 18%. For 100 million ether, the return rate drops to 1.81%. This is not a cause of concern, however, because locked ether is taken out of circulation and given that it's "fuel" for the network, the price of the asset should rise proportionately as demand increases, in particular due to other [lock-ups in DeFi](https://defipulse.com/).
+
+### Slashing
+
+Reasons for slashing are the same in both systems:
+
+- producing two blocks for the same slot
+- signing two conflicting chains
+- being offline
+
+Additionally, in Polkadot there's the penalty for illegal randomness rolls which can be detected by fishermen. See Randomness section above for details.
+
+In Polkadot any offence not only slashes, but also removes a validator from the validator set in the current era. They have to re-qualify to rejoin.
+
+## Smart Contracts, Execution Environments, Runtimes and SPREE
+
+Finally, let's talk bout the meat of the protocol - the execution layer.
+
+In Ethereum, there will be the concept of Execution Enivronments. These will be special runtimes built by special teams and funded by special interests to run and execute custom code. In other words, if someone wants to build the ability to process bitcoin-like transactions into Ethereum, they need to fund a UTXO-execution-environment and make sure all clients add it to their code as well (likely helping with funding those efforts, or contributing outright). Once done, the execution environment becomes available to everyone - it's considered a network upgrade.
+
+The first and default execution environment will be eWASM, a neutered version of WASM flavored for Ethereum. The second one may very well be an EVM EE which would allow Ethereum 1 to continue living and breathing inside of Ethereum 2, perhaps on a dedicated shard. Developing custom EEs will be expensive and hard, so we don't expect to see many of them.
```

Comment:
```suggestion
The first and default execution environment will be ewasm, a neutered version of Wasm flavored for Ethereum. The second one may very well be an EVM EE which would allow Ethereum 1 to continue living and breathing inside of Ethereum 2, perhaps on a dedicated shard. Developing custom EEs will be expensive and hard, so we don't expect to see many of them.
```

Reply:
Strangely, the correct capitalization is "Wasm" but also "ewasm".

https://webassembly.org/

https://github.com/ewasm/design/commit/16c31a57c35fdd026b8238a8e63ddd9ef3c21918